hepcrawl/extractors/jats.py

```diff
@@ -68,6 +68,16 @@ def _get_keywords(self, node):
                     free_keywords.append(keyword)
         return free_keywords, classification_numbers
 
+    def _get_public_notes(self, node):
+        """Return list of public notes from node."""
+        public_notes = []
+        for note in node.xpath('//notes[@notes-type]'):
+            for keyword in note.xpath('p/text()').extract():
+                public_notes.append({
+                    'value': keyword,
+                })
+        return public_notes
+
```

Comment:
With https://github.com/inspirehep/hepcrawl/pull/112 and https://github.com/inspirehep/hepcrawl/pull/115 we have the extra tests at the same level as arxiv so we can continue on this PR.